cmd/syncv3/main.go

```diff
@@ -112,6 +113,16 @@ func main() {
 				panic(err)
 			}
 		}()
+		// Cribbed from synapse
+		// https://github.com/matrix-org/synapse/blob/d93912042191d30ff1f7aa41d9f0779a609caca8/synapse/metrics/__init__.py#L413-L421
+		// and https://www.robustperception.io/exposing-the-software-version-to-prometheus/
+		versionGauge := prometheus.NewGauge(prometheus.GaugeOpts{
+			Namespace:   "sliding-sync",
+			Name:        "build-info",
```

Comment:
These should be underscores. If not, runtime error.

Need to enable prometheus metrics endpoint in the E2E tests.

Reply:
We have an entire file called metrics_test.go in integration which do this.

Reply:
This could be as trivial as starting the binary with the SYNCV3_PROM env var set then ensuring /metrics 200 OKs.